packages/core/index.html

```diff
@@ -58,9 +60,19 @@ <h1>My App2</h1>
       </ul>
     </div>
   </div>
+  <div id="extra"></div>
 </body>
 
 <script type="text/javascript">
-  document.querySelectorAll(".click").forEach(elem =>
-    elem.addEventListener("click", e => { console.log("clicked", e.currentTarget) }))
+  document.querySelectorAll(".click").forEach((elem) =>
+    elem.addEventListener("click", (e) => {
+      // console.log("clicked", e.currentTarget);
+    })
+  );
+
+  document.querySelector("#two").addEventListener("click", (e) => {
+    const newElem = document.createElement("div");
+    newElem.innerHTML = "<p>abcd</p>";
```

Comment:
you could have written here something nice (or at least "hello world")

Reply:
Yes I could have :)

Reply:
Treating it as kind of a throwaway file at the moment, we can make it nicer.